packages/components/select/__tests__/select.spec.ts

```diff
@@ -1521,6 +1521,18 @@ describe('Select', () => {
     expect(innerInputEl.placeholder).toBe(placeholder)
   })
 
+  test('should close popper when click icon twice', async () => {
+    wrapper = getSelectVm({
+      filterable: true,
+      clearable: true,
+    })
+    const select = wrapper.findComponent({ name: 'ElSelect' })
+    const suffixIcon = select.find('.el-input__suffix')
+    await suffixIcon.trigger('click')
```

Comment:
We should assert the popper is visible after one click to make the result more reliable.